test/semantics/test_symbols.sh

```diff
@@ -28,13 +28,15 @@ src2=$temp/2.f90
 src3=$temp/3.f90
 diffs=$temp/diffs
 
-# Strip out blank lines and all comments except "!DEF:" and "!REF:"
-sed -e 's/!\([DR]EF:\)/KEEP \1/' \
+# Strip out blank lines and all comments except
+# "!DEF:", "!REF:", "!$omp", "c$omp", and "*$omp"
+sed -e 's/!\([DR]EF:\)/KEEP \1/' -e 's/[!c*]\($omp\)/KEEP \1/' \
   -e 's/!.*//' -e 's/ *$//' -e '/^$/d' -e 's/KEEP \([DR]EF:\)/!\1/' \
+  -e 's/KEEP \($omp\)/!\1/' \
```

Comment:
This turns `c$omp` and `*$omp` into `!$omp`. Is that okay? What do those even mean?

Reply:
Yes. OpenMP directive accepts fixed forms (`!$omp`, `c$omp`, and `*$omp`; starts with column 1) and free forms (`!$omp`; mostly used). For testing purposes, all `1.f90`, `2.f90`, and `3.f90` will be using the same free form (`!$omp`), which is okay.

Reply:
I don't see the point of allowing `c$omp` and `*$omp` in the input when it's just going to be changed to `!$omp` anyway. Why not just require `!$omp`?

Reply:
Yeah, that makes sense. I will remove that.